app/javascript/gobierto_data/webapp/components/sidebar/SidebarDatasets.vue

```diff
@@ -95,17 +107,27 @@ export default {
   },
   methods: {
     orderDatasets() {
-      const sortDatasets = this.items
-      const allDatasets = sortDatasets.sort((a, b) => a.attributes.name.localeCompare(b.attributes.name));
-      let slug = this.$route.params.id
-      const indexToggle = allDatasets.findIndex(dataset => dataset.attributes.slug == slug)
+      //TODO Datasets should be order in filter mixin
+      const allDatasets = this.items.sort((a, b) => a.attributes.name.localeCompare(b.attributes.name));
```

Comment:
Same props destructuration means:
```suggestion
      const allDatasets = this.items.sort(({ attributes: { name: a } = {} }, { attributes: { name: b } = {} }) => a.localeCompare(b));
```